katello/katello/hostname-change.rb

```diff
@@ -331,8 +331,11 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' /etc/foreman-installer/scenarios.d/*.yaml")
 
-      STDOUT.puts "removing last_scenario.yml file"
-      self.run_cmd("rm -rf /etc/foreman-installer/scenarios.d/last_scenario.yaml")
+      STDOUT.puts "backing up last_scenario.yaml"
+      self.run_cmd("cp #{last_scenario_yaml} /var/tmp")
```

Comment:
I'd prefer some ```mktemp``` based solution rather than static ```/var/tmp```. Also a check if the last_scenario file actually exists might be a good one, even if the chance is of issues is low.